tests/forms/gridfield/GridFieldAddExistingAutocompleterTest.php

```diff
@@ -111,3 +111,16 @@ public function Form() {
 		return new Form($this, 'Form', new FieldList($field), new FieldList());
 	}
 }
+
+class GridFieldTest_Team extends DataObject implements TestOnly {
```

Comment:
This class already exists, it seems. Either you need to pick a new name or you should just amend the existing class to have the canView function you need (assuming that doesn't break other tests)!